### ISSUE TITLE
KHMX8 bad operation destription

### ISSUE THREAD

**Issue by marcfedorow** [riscv/riscv-p-spec#65]
```
if (is “KHM8”) {
  op1t = Rs1.B[x+1]; op2t = Rs2.B[x+1]; // top
  op1b = Rs1.B[x];   op2b = Rs2.B[x];   // bottom
} else if (is “KHMX8”) {
  op1t = Rs1.H[x+1]; op2t = Rs2.H[x];   // Rs1 top
  op1b = Rs1.H[x];   op2b = Rs2.H[x+1]; // Rs1 bottom
}
```
I suppose RsX.B must stay instead of RsX.H

**Comment by chuanhua** [comment]
Thanks for finding this issue. I will fix this typo in the next release.

**Comment by marcfedorow** [comment]
`For the “KHMX16” instruction, multiply the top 8-bit Q7 content of 16-bit chunks in Rs1 with the bottom 8-bit Q7 content of 16-bit chunks in Rs2. At the same time, multiply the bottom 8-bit Q7 content of 16-bit chunks in Rs1 with the top 8-bit Q7 content of 16-bit chunks in Rs2.`
I believe there should stay `For the “KHMX8” instruction`.

**Comment by chuanhua** [comment]
Thanks. I will fix it in the next release.

**Comment by chuanhua** [comment]
Fixed in 7ca35508bf5.
